opencda/core/sensing/prediction/physics.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+"""
+Physcis-based trajectory prediction model
```

Comment:
Restructure this core script please. Rename it to physics_model, and put all functions that are a member of a class to another script(suggested name: physics_utils). By this way, the code structure is more clear.